### PR TITLE
Fix Using / for division is deprecated

### PR DESCRIPTION
**Pull request by forxer**
This PR intend to solve #4015

### CODE REVIEW

File: build/scss/plugins/_select2.scss [hunk 1]
```diff
@@ -241,11 +241,11 @@ select.form-control-sm ~ {
       min-height: $input-height-sm;
 
       .select2-selection__rendered {
-        padding: 0 $input-padding-x-sm / 2 $input-padding-y-sm;
-        margin-top: -($input-padding-x-sm / 5);
+        padding: 0 $input-padding-x-sm * .5 $input-padding-y-sm;
+        margin-top: -divide($input-padding-x-sm, 5);
```
Comment (danny007in) [inline build/scss/plugins/_select2.scss:245]: fix this `-divide(`

**Review by danny007in — CHANGES_REQUESTED** [review]

File: build/scss/plugins/_select2.scss (hunk 1 above)
Comment (forxer) [reply build/scss/plugins/_select2.scss:245]: I'm not sure what I need to fix this is working for me. Can you tell me more? I would be happy to correct it.

Comment (forxer) [reply build/scss/plugins/_select2.scss]: ok done

**Comment by REJack** [comment]
There are still bug, please fix them then I can merge it 😄

**Comment by forxer** [comment]
I do not understand why the tests do not pass. I replaced :

`height: $main-header-height / 2;`

through :

`height: $main-header-height * 0.5;`

It's valid, that's what they did on [Bootstrap in this PR ](https://github.com/twbs/bootstrap/pull/34571/files)

**Comment by REJack** [comment]
@danny007in do you have any clue about the errors?

**Comment by REJack** [comment]
@forxer Could you enable the edit rights for maintainers? Then I would look over it tonight (CET) 😄

**Pull request merged**